packages/deployments/contracts/contracts_forge/Connext.t.sol

```diff
@@ -12,26 +16,83 @@ import "../contracts/ProposedOwnableUpgradeable.sol";
 // other forge commands: yarn workspace @connext/nxtp-contracts forge <CMD>
 // see docs here: https://onbjerg.github.io/foundry-book/index.html
 
+contract TestDummyBridgeRouter {
+  function send(
+    address _token,
+    uint256 _amount,
+    uint32 _destination,
+    bytes32 _recipient,
+    bool _enableFast,
+    bytes32 _externalHash
+  ) external {}
+}
+
 contract ConnextTest is ForgeHelper {
   // ============ Libraries ============
   using stdStorage for StdStorage;
 
   event MaxRoutersPerTransferUpdated(uint256 maxRouters, address caller);
+  event XCalled(
+    bytes32 indexed transferId,
+    address indexed to,
+    IConnext.CallParams params,
+    address transactingAsset,
+    address localAsset,
+    uint256 transactingAmount,
+    uint256 localAmount,
+    uint256 relayerFee,
+    uint256 nonce,
+    address caller
+  );
```

Comment:
will we always have to define our own events in test contracts like this or is there a better way to import them from the source?

Reply:
Unfortunately, it seems there is no way of importing them